lib/recurly/errors.js

```diff
@@ -316,12 +316,9 @@ function recurlyErrorFactory (definition) {
     }
 
     if (this.reporter instanceof Reporter) {
-      let type = 'client';
-      if (this.classification) type += `:${this.classification}`;
```

Comment:
`classification` was always undefined here and the reporter would always send `type: 'client'`, regardless of the error.